commands/command.go

```diff
@@ -38,3 +42,22 @@ func Register(bot *eelbot.Bot, opts map[string]any, db *sql.DB) error {
 	}
 	return nil
 }
+
+func queryRow(db *sql.DB, dbTimeout time.Duration, query string, args ...any) (*sql.Row, context.CancelFunc) {
+	ctx, cancel := context.Background(), func() {}
+	if dbTimeout > 0 {
+		ctx, cancel = context.WithTimeout(ctx, dbTimeout)
+	}
+	return db.QueryRowContext(ctx, query, args...), cancel
+}
+
+// Returns a query to select a random row from a table that contains an integer primary key column called 'id' that
+// maintains gapless sequential values. This is much more performant than "ORDER BY RANDOM()" since it doesn't require
```

Comment:
Should ensure eelbot-ui and w/e else does the data mgmt side of eelbot actually does reindex if it deletes items

Reply:
Yes, it should.